.bazelrc

```diff
@@ -0,0 +1,22 @@
+build --cxxopt=-Wno-format
+build --cxxopt=-std=c++14
+build --cxxopt=-fdiagnostics-color=always
+build --cxxopt=-Wno-noexcept-type
+build --cxxopt=-Wall
+build --cxxopt=-Wextra
+build --cxxopt=-march=broadwell
+build --cxxopt=-fPIC
+build --cxxopt=-fno-builtin-malloc
+build --cxxopt=-fno-builtin-calloc
+build --cxxopt=-fno-builtin-realloc
+build --cxxopt=-fno-builtin-free
+build --cxxopt=-fno-omit-frame-pointer
+build --cxxopt=-Wno-unused-parameter
+build --cxxopt=-DGOOGLE_PROTOBUF_NO_RTTI
+build --cxxopt=-Wno-unused-result
+build --cxxopt=-DNDEBUG
+build --cxxopt=-g
```

Comment:
Thank you for sending us your PR! 
If I understand correctly, `bazel` supports debug and release modes in c/c++.
```
build --cxxopt=-DNDEBUG
build --cxxopt=-g
build --cxxopt=-O
```
is something that usually defined by such modes.

Reply:
in bazel it's called "compilation modes (opt, dbg, fastbuild)" but you need to check how to pass them. For each mode a different build directory is created....

Reply:
Also, I think the correct way to set compiler flags is to use features and actions but I am having hard time to find good example. 

https://github.com/bazelbuild/bazel/issues/5198

https://docs.bazel.build/versions/0.26.0/cc-toolchain-config-reference.html#example-usage

and this example: 
https://github.com/mjbots/rules_mbed/blob/f2e97268b0ca2f214892753ceb914773b79daf32/tools/cc_toolchain/cc_toolchain_config.bzl#L332

Though I do not understand much here

Reply:
Will change to features.

Reply:
@romange So, after some reading, I removed the DBG flags from the bazelrc, they're automatically added when you build with `--compilation_mode=dbg`.
The rest of the flags are hard coded in CMAKE so I'm keeping them in the file, but if in the future we want to separate, the bazel way is to use `config_setting` and `select` to defined those value.

Are you ok with that? or still requires some changes?